docs/data-management-on-carbondata.md

```diff
@@ -283,7 +283,30 @@ This tutorial is going to introduce all commands and data operations on CarbonDa
 	 ```
 	 ALTER TABLE employee SET TBLPROPERTIES (‘CACHE_LEVEL’=’Blocklet’)
 	 ```
-	 
+
+   - **String longer than 32000 characters** (Alpha Feature 1.4.1)
+
+     In common scenarios, the length of string is less than 32000, so carbondata store the length of content using Short to reduce memory and space consumption.
+     If your string content exceeds the 32000 characters limitation, you can specify the columns as 'long string column' using below tblProperties:
```

Comment:
comparing to previous paragram, better to describe we are using int32 to store length of the long string